plugins/hydra_ax_sweeper/tests/config/params/nested_with_escape_char.yaml

```diff
@@ -1,8 +1,8 @@
 # @package hydra.sweeper.ax_config.params
-quadratic_x\_arg:
+quadratic.x_arg:
```

Comment:
Should this be deleted? the file name no longer represent what this is doing.

Reply:
Ouch my bad. Thanks for spotting.